UIKitMovie/UIKitMovieTests/AssemblyBuilderTests.swift

```diff
@@ -0,0 +1,35 @@
+// AssemblyBuilderTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import UIKitMovie
+import XCTest
+
+/// Unit-тест сборщика экранов
```

Comment:
```suggestion
///  Тестирование сборщика экранов
```
Unit-тест и так понятно